### PR TITLE
Echoes: Logic Updates and Video Upload

### PR DESCRIPTION
**Pull request by CMos75**
#### logic database

- Added: 306 videos to the logic database. see the [Video Directory] (https://randovania.github.io/Metroid%20Prime%202%20Echoes/) for the full collection

##### Temple Grounds:

- Added:  NSJ Extended Dash (Expert) to cross Grand Windchamber through the middle platform.

##### Agon Wastes:

- Added: NSJ Extended Dash (Advanced) to reach Temple Access Door in Mining Station A.

##### Sky Temple Ground:

- Removed: Phazon Grounds NSJ, No SA -> Invisibil Objects (Hypermode) or Movement (Expert) and Dark Visor. Doesn't exist.

##### Sanctuary Fortress:

- Added: Extended Dash (Expert) to reach the Scan Post in Watch Station Access from Main Gyro Chamber Door.
- Added: Extended Dash (Expert) to reach Main Gyro Chamber Door in Watch Station Access from the Scan Post Side.

### CODE REVIEW

File: randovania/games/prime2/json_data/Temple Grounds.txt [hunk 1]
```diff
@@ -395,18 +428,26 @@ Extra - in_dark_aether: False
...
+          All of the following:
+              # https://www.youtube.com/watch?v=JRT1SEeWKv8
+              Screw Attack into Tunnels/Openings (Intermediate) and Use Screw Attack (Space Jump)
           All of the following:
               Morph Ball
               Any of the following:
-                  Space Jump Boots and Instant Morph (Intermediate)
-                  Morph Ball Bomb and Bomb Space Jump (Expert) and Instant Morph (Expert)
+                  All of the following:
+                      # https://www.youtube.com/watch?t=17&v=JRT1SEeWKv8&feature=youtu.be
```
Comment (henriquegemignani) [inline randovania/games/prime2/json_data/Temple Grounds.txt:438]: This causes issues with the code that generates the video directory.

**Review by henriquegemignani — CHANGES_REQUESTED** [review]

**Comment by henriquegemignani** [comment]
@dyceron can you take a look, particularly the difficulty changes

**Review by dyceron — CHANGES_REQUESTED** [review]
I stopped pointing out each case, but saying "Use Double Bomb Jump" as a comment is weird and should just say "Double Bomb Jump" since it still gets the point across and more correct.

File: randovania/games/prime2/json_data/Temple Grounds.txt [hunk 2]
```diff
@@ -1769,7 +1769,7 @@ Extra - in_dark_aether: True
           Any of the following:
               Morph Ball Bomb
               All of the following:
-                  # Power Bombs to exit Boost Ball Spinners
+                  # Use Power Bombs to exit Boost Ball Spinners
```
Comment (dyceron) [inline randovania/games/prime2/json_data/Temple Grounds.txt:1772]: Add this comment to the yellow Seeker lock path as well

**Review by dyceron — CHANGES_REQUESTED** [review]
Also fix the changelog conflicts

**Pull request merged**